app/uk/gov/hmrc/excisemovementcontrolsystemapi/config/AppConfig.scala

```diff
@@ -24,10 +24,13 @@ import scala.concurrent.duration.{DAYS, Duration, FiniteDuration, HOURS, MINUTES
 
 @Singleton
 class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig) {
-
   val appName: String = config.get[String]("appName")
 
   lazy val eisHost: String = servicesConfig.baseUrl("eis")
+  lazy val nrsHost:  String = servicesConfig.baseUrl("nrs")
+
+  lazy val nrsAuthorisationToken: String = servicesConfig.getString("microservice.services.nrs.api-key")
```

Comment:
You can get a string from the servicesConfig without putting the `microservice.services` root folder at the start and just using `nrs.api-key`

Reply:
done